packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -604,64 +604,60 @@ export class SDKClient {
     requestId?: string,
   ): Promise<T> {
     const requestIdPrefix = formatRequestIdMessage(requestId);
-    const currentDateNow = Date.now();
+    const queryType = query.constructor.name;
+    let queryResponse: any = null;
+    let queryCost: number | undefined = undefined;
```

Comment:
Nit: Does this need to be initialized to `undefined`?

Reply:
Ah yeah its a good point but I initilized to undefined as later the queryCost is assigned to `query._queryPayment?.toTinybars().toNumber()` which can be a number or undefined.